contracts/lifecycle/TimeFrame.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity ^0.5.0;
```

Comment:
We've upgraded to 0.5.2 in #1606 (mainly to reduce false positive warnings)

Reply:
changed to 0.5.2